test/NuGet.Core.Tests/NuGet.Packaging.Test/ReferencesInNuspecMatchRefAssetsRuleTests.cs

```diff
@@ -0,0 +1,377 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Castle.DynamicProxy.Generators.Emitters.SimpleAST;
+using NuGet.Common;
+using NuGet.Packaging.Rules;
+using Xunit;
+using static NuGet.Packaging.Rules.ReferencesInNuspecMatchRefAssetsRule;
+
+namespace NuGet.Packaging.Test
+{
+    public class ReferencesInNuspecMatchRefAssetsRuleTests
+    {
+        [Fact]
+        public void Compare_PackageWithReferencesAndRefFolderMatching_ShouldBeEmpty()
+        {
+            //Arrange
+            var references = new Dictionary<string, IEnumerable<string>>
+            {
+                {"net462", new List<string>() {"MyLib.dll", "MyHelpers.dll"} }
```

Comment:
minor: formatting. Plus on other places in this file where this line was copy/pasted.

Reply:
What kind of formatting should be doing here?

Reply:
if you use Visual Studio's "format selection" or "format document", it will automatically adjust it. You can also apply default formatting to code blocks by deleting the `}` and then typing it again. There is usually whitespace around `{` characters.